op-deployer/book/src/reference-guide/releases.md

```diff
@@ -0,0 +1,92 @@
+# Releases
+
+## Versioning
+
+For all releases after `v0.0.11`, each minor version of OP Deployer will support the current governance-approved
```

Comment:
Not too concerned about the version numbers, but should we just make op-deployer v1.0.0 corresponding with the new v2.0.0 version of the contracts?

Also, should an op-deployer tag _always_ be cut at the same time as op-contracts tags? I'm wondering if we should just merge the two processes into a single tag/release. We need to update the contract versioning doc, but it's here: https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/meta/VERSIONING.md

Reply:
I'd like to keep it prerelease as much as possible - Go's [tagging requirements](https://go.dev/doc/modules/major-version) start becoming really painful once you introduce a major version. Keeping it prerelease will help us move fast.

> Also, should an op-deployer tag always be cut at the same time as op-contracts tags?

Usually, but not necessarily. We can discuss whether we should merge the processes further offlien.